File: f77/build05/drvrdata.f
C
      Subroutine Driver_Default_Data
C
C     This subroutine sets the default values for the user-accessable
C     system variables.
C
C     Inputs:
C        None.
C
C     Outputs:
C        Dt    - Integration step size [sec]
C        Time  - Simulation time [sec]
C        Time0 - Initial time [sec]
C        TStop - Simulation termination time [sec]
C
C     Internal variables and constants:
C        None.
C
C     Declare global common
C
      Include '../driver/global.inc'
      Include '../driver/sysvars.inc'
C
C     Set system default values
C
      Time   = 0.0e0
      Time0  = 0.0e0
      TStop  = 0.0e0
      Dt     = 0.00e0
C
      Return
      End

File: compcert/driver/drvrdata.c
#include "../driver/global.h"
#include "../driver/sysvars.h"
void driver_default_data() {
  // This subroutine sets the default values for the user-accessable
  // system variables.
  // Set system default values
  time    = 0.0;
  time0   = 0.0;
  tstop   = 0.0;
  dt      = 0.005;
  dtmax   = 0.005; // Maximum allowable simulation time step [sec]
  dtmin   = 0.005; // Minimum allowable simulation time step [sec]
  dtprint = 0.01;  // Time step between printing data [sec]
  reeval_derivs = FALSE;
}
